packages/@react-aria/overlays/docs/useModal.mdx

```diff
@@ -0,0 +1,33 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/overlays';
+import {HeaderInfo} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+---
+category: Overlays
+keywords: [dialog, modal, tray, popover, aria]
+---
+
+# useModal
+
+<p>{docs.exports.useModal.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useModal']} />
+
+## Example
```

Comment:
missing an API section?

Reply:
useModal actually doesn't have any args and doesn't return anything so the FunctionAPI component doesn't render, lemme see if I can change it so it shows that

Reply:
haha, great, yeah, we should probably still show that if possible

Reply:
nvm I'm dumb, forgot to import it zzzzzz